src/brain_teasers/python/add_string_numbers/src/add_string_numbers.py

```diff
@@ -0,0 +1,19 @@
+#Add string numbers
+#Given two strings of numbers, add the numbers together and return the result as a string.
+#Example:
+#Input: "123", "456"
+
+
+def add_string_numbers(num1, num2):
+    # Fill this in.
+    return str(int(num1) + int(num2))
```

Comment:
I tested your code and got two errors. Your code only with integers, it doesn't work with floats.